cookbooks/simple_ice_box.prm

```diff
@@ -0,0 +1,174 @@
+# A simple setup for using the boundary heat flux in a ice viscoplatic model.
+
+
+set Dimension                              = 2
+set Use years in output instead of seconds = true
+set End time                               = 1e6
+set Output directory                       = output-ice-box_2d
+set Nonlinear solver scheme                = single Advection, iterated Stokes 
+set Nonlinear solver tolerance             = 1e-5
+set Max nonlinear iterations               = 10
+set CFL number                             = 0.5
+set Pressure normalization                 = no
+set Use conduction timestep                = true
+# Material model
+# Rheology: Non-linear viscous flow and Drucker Prager Plasticity
+# Values for most rheological parameters are specified for a background material and
+# each compositional field.  Values for viscous deformation are based on diffusion creep 
```

Comment:
Should `diffusion creep` be changed to `dislocation creep` here?

Reply:
It may also be helpful to say how you scaled the prefactor value from unixial to trixial stress (or at least say this was done).